assembl/static2/css/components/posts.scss

```diff
@@ -141,7 +141,7 @@
 
     .body {
         padding: 20px 0;
-        word-break: break-all;
+        word-wrap: break-word;
```

Comment:
OK yes it seems that IE and Edge still need the old name http://caniuse.com/#search=overflow-wrap